packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv6/ipv6.controller.js

```diff
@@ -1,5 +1,5 @@
 import find from 'lodash/find';
-
+import ovhManagerRegionService from '@ovh-ux/manager-components/src/region/service';
```

Comment:
The import should be defined in the ipv6 module file and `ovhManagerRegionService` should be injected in the constructor like other services

```
import { region } from '@ovh-ux/manager-components';

const moduleName = 'ovhManagerDedicatedAgoraOrderIpv6';
angular
  .module(moduleName, [
    managerCore,
    ngOvhUtils,
    'oui',
    'ovh-api-services',
    'pascalprecht.translate',
    'ui.router',
    'ovhManagerCatalogPrice',
   region
  ])

```